pandas/tests/io/formats/test_to_latex.py

```diff
@@ -199,72 +197,69 @@ def test_to_latex_midrule_location(self):
 class TestToLatexLongtable:
     def test_to_latex_empty_longtable(self):
         df = DataFrame()
-        result = df.to_latex(longtable=True)
+        result = df.to_latex(environment="longtable")
```

Comment:
Looks like ``longtable=True`` will not be working now.
I mean you issue a warning if ``longtable`` is not None, but would it not be more reasonable to also ensure that if a user supplied ``longtable=True`` the longtable is actually created.

Reply:
Technically I could try to work around the existing args feeding the new ones, but for this first review I wanted to get some feedback and the cleanest at the start was to simply raise a warning that it has been replaced.